perception/autoware_image_projection_based_fusion/config/mask_cluster_fusion.param.yaml

```diff
@@ -0,0 +1,5 @@
+/**:
+  ros__parameters:
+    fusion_distance: 100.0 # the distance threshold for the fusion
+    fusion_ratio: 0.4 # the ratio between the selected cluster points and the total points in the cluster
+    remove_unknown: true # remove unknown clusters
```

Comment:
Please add JSON schema.

Reply:
Missing json schema is added.

related commit: https://github.com/autowarefoundation/autoware.universe/pull/9482/commits/7ef4ab54ed1439bde6ab45cb6ff50da1a181f100